### PR TITLE
TBQ23: Update readme.md

### PR DESCRIPTION
**Pull request by AVM82**

### CODE REVIEW

**Comment by Sekator778** [comment]
all done

**Review by Tsurkan — APPROVED** [review]
Ok

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# Getting Started
\ No newline at end of file
+# Getting Started
+### Git
+
+#### Branches
+
+- **`main`** - production source code.
+- **`development`** - staging source code.
```
Comment (Sekator778) [inline README.md:7]: need more info

**Review by Sekator778 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AVM82) [reply README.md:7]: hi

**Pull request merged**